arbor/partition_load_balance.cpp

```diff
@@ -188,7 +188,7 @@ domain_decomposition partition_load_balance(
                 group_elements.push_back(cell.id);
             } else {
                 if (group_elements.size() + super_cells[cell.id].size() > group_size && !group_elements.empty()) {
-                    groups.push_back({k, std::move(group_elements), backend});
+                    groups.push_back({k, group_elements, backend});
```

Comment:
This one is a false positive from static analysis: we clear `group_elements` after move, so that it is in a valid and known (empty) state.

Reply:
Correct, for STL containers and `clear` this is ok. I'll revert this.